airbyte-integrations/connectors/source-monday/unit_tests/integrations/test_teams_stream.py

```diff
@@ -0,0 +1,135 @@
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+
+import json
+from unittest import TestCase
+from unittest.mock import patch
+
+from airbyte_cdk.test.mock_http import HttpMocker
+from airbyte_protocol.models import Level as LogLevel
+from airbyte_protocol.models import SyncMode
+
+from .config import ConfigBuilder
+from .monday_requests import TeamsRequestBuilder
+from .monday_requests.request_authenticators import ApiTokenAuthenticator
+from .monday_responses import ErrorResponseBuilder, TeamsResponseBuilder
+from .monday_responses.records import TeamsRecordBuilder
+from .utils import get_log_messages_by_log_level, read_stream
+
+
+class TestTeamsStreamFullRefresh(TestCase):
```

Comment:
Need change bypass reason in CAT config and mention that this stream has no tests data and tested with integration tests.